src/sftp/handler.go

```diff
@@ -21,18 +21,19 @@ type SftpHandler struct {
 	blobHandler      usecases.BlobHandler
 	credentialGetter secrets.CredentialGetter
 	zipHandler       zip.ZipHandlerInterface
+	partnerId        string
 }
 
-func NewSftpHandler(credentialGetter secrets.CredentialGetter) (*SftpHandler, error) {
+func NewSftpHandler(credentialGetter secrets.CredentialGetter, partnerId string) (*SftpHandler, error) {
 	// In the future, we'll pass in info about what customer we're using (and thus what URL/key/password to use)
```

Comment:
I like that idea, are these future details we need to add to this future card: https://flexion.slack.com/archives/C055XTF22B0/p1735853644609399

Reply:
This was added to the Eng Task list.